### PR TITLE
Style Devise forms (#58)

### LINKED ISSUE #58: Style Devise forms (register, login)
- [ ] Sign up #15
- [ ] Sign in #16
- [ ] Edit profile

Depends on #52.

### PR DESCRIPTION
**Pull request by neonowy**
* Modified original Devise forms to work with Hocus-Pocus.
* Added styles.

Closes #58.

### CODE REVIEW

**Comment by pospieszynski** [comment]
:+1: 
@gotar

File: app/assets/stylesheets/_devise.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.sessions, .registrations, .passwords {
```
Comment (smt116) [inline app/assets/stylesheets/_devise.scss]: I would put each class into separate line:

```
.sessions,
.registrations,
... {
}
```

File: app/assets/stylesheets/_forms.scss [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+    text-align: center;
+    color: $color-gray;
+  }
+
+  form {
+    margin: 0;
+  }
+
+  #error_explanation {
```
Comment (smt116) [inline app/assets/stylesheets/_forms.scss]: why with `_` instead of `-`?

Comment (neonowy) [inline app/assets/stylesheets/_forms.scss]: @smt116 It's default devise ID in `devise_error_messages!` but I can override it if you want.

Comment (gotar) [inline app/assets/stylesheets/_forms.scss]: leave it it's only few days to the end, better focus all power to finish product on time, it's not important

**Comment by gotar** [comment]
looks ok you can merge

**Pull request merged**